votor-messages/src/migration.rs

```diff
@@ -40,3 +54,244 @@ pub const MIGRATION_MALICIOUS_THRESHOLD: f64 = 20.0 / 100.0;
 ///
 /// Using 38% as the `SWITCH_FORK_THRESHOLD` gives us 82% for `GENESIS_VOTE_THRESHOLD`.
 pub const GENESIS_VOTE_THRESHOLD: f64 = 82.0 / 100.0;
+
+/// The interval at which we refresh our genesis vote
+pub const GENESIS_VOTE_REFRESH: Duration = Duration::from_millis(400);
```

Comment:
intentionally set to slot time? Should we derive it?

Reply:
1) nixing gossip completely 🪓 
2) My understanding was that the 2k capped validator set / 1.6 sol per epoch VAT would roll out first (wen has it keyed as a separate feature flag). At that point we could run an A2A test, but not sure there's any value considering we're running it on testnet rn uncapped?